src/main/fc/fc_msp.c

```diff
@@ -452,7 +452,7 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
 
             if (cmdMSP == MSP_STATUS_EX) {
                 sbufWriteU16(dst, averageSystemLoadPercent);
-                sbufWriteU16(dst, armingFlags);
+                sbufWriteU16(dst, armingDisableFlags);
```

Comment:
This is a breaking change. The `ARMED` bit of this field is used by external devices to detect whether the aircraft is in the armed state.

Reply:
I suggest the FIXME should be forked out of this PR into a separate one for further discussion.